docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -4,48 +4,55 @@ description: List of operations that users can perform on Codacy depending on th
 
 # Roles and permissions for organizations
 
-Your team members have different permission levels on Codacy depending on their role on your Git provider. To change the permission level of a user on Codacy, you must adjust their role directly on your Git provider so that Codacy will use the corresponding permission level on the next time that the user logs in to Codacy.
+By default, Codacy assigns each organization member a role corresponding to that member's role on your Git provider.
 
-See the Codacy permission levels that correspond to each role on your Git provider:
+To update a member's role on Codacy, update that member's role directly on your Git provider. When next logging in to Codacy, the member is assigned the new role.
+
+To review the permissions granted by each role, see the tables for each Git provider:
 
 -   [GitHub](#permissions-for-github)
 -   [GitLab](#permissions-for-gitlab)
 -   [Bitbucket](#permissions-for-bitbucket)
 
-See [managing people](managing-people.md) to list and manage the members of your Codacy organization.
+You can grant some administrative permissions to any organization member independently of the member's role on the Git provider, using the [organization manager](#the-organization-manager-role) role.
```

Comment:
@nicklem Nice, you dropped the tip! 👍 
Leaving a small tweak to pass the idea that "we listed the default roles above, but we also have this other one". Let me know if you agree. 

```suggestion
Additionally, you can grant some administrative permissions to any organization member independently of the member's role on the Git provider, using the [organization manager](#the-organization-manager-role) role.
```